src/Components/ImagesTable/Release.tsx

```diff
@@ -26,6 +26,7 @@ const Release = ({ release }: ReleaseProps) => {
     'rhel-94': 'RHEL 9.4',
     'centos-8': 'CentOS Stream 8',
     'centos-9': 'CentOS Stream 9',
+    'centos-10': 'CentOS Stream 10',
```

Comment:
should it be at a seperate pr?

Reply:
This is a change that's coming with the API schema update. Since centos-10 was added to the API, it needs to be also added to Release.tsx otherwise TypeScript would be very unhappy.